hash.c

```diff
@@ -1857,27 +1860,35 @@ rb_hash_initialize(int argc, VALUE *argv, VALUE hash)
  *  but the argument is not an array of 2-element arrays or a
  *  {Hash-convertible object}[doc/implicit_conversion_rdoc.html#label-Hash-Convertible+Objects]:
  *
- *    Hash[:foo] # Raises ArgumentError (odd number of arguments
- *    Hash[ [ [:foo, 0, 1] ] ] # Raises ArgumentError (invalid number of elements (3 for 1..2))
+ *    # Raises ArgumentError (odd number of arguments
```

Comment:
Is this missing a terminating bracket?

Reply:
That's affirmative.

Reply:
Once you are done with the feedback you can click "Resolve Issue". I have no problem with you taking the lead on that.

Reply:
I checked what is actually generated: `ArgumentError (odd number of arguments for Hash)` so we can use this.

Reply:
Fixed.  (And thanks for correcting my careless second error.)